shared/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gcp_jupyterlab_shared",
-  "version": "1.0.6",
+  "version": "1.0.7",
```

Comment:
Please bump the version for the Python side as well (even though there were no changes there) to keep them in sync.